lib/element-helper-syntax-plugin.js

```diff
@@ -48,16 +48,24 @@ function transformParts(node, b) {
     // path
     path: b.path('component', node.path.loc),
     // params
-    params: [b.sexpr(
-      // path
-      b.path('-element', node.path.loc),
-      // params
-      node.params,
-      // hash
-      node.hash,
-      // loc
-      node.loc
-    )],
+    params: [
+      b.sexpr(
+        // path
+        b.path('ensure-safe-component', node.path.loc),
```

Comment:
Hmm. Where does this come from when the app is not being built by embroider? (`@embroider/macros` is only a devDep)

Reply:
This is provided by `@embroider/util`, which is a "real" dependency!